src/main/java/uk/gov/digital/ho/hocs/info/api/TeamResource.java

```diff
@@ -148,6 +151,26 @@ public ResponseEntity deleteTeamPermissions(@PathVariable String teamUUID, @Requ
         return ResponseEntity.ok(teams.stream().map(TeamDto::from).collect(Collectors.toSet()));
     }
 
+    @GetMapping(value = "/team/users")
+    public ResponseEntity<Set<UserWithTeamsDto>> getUsersForTeam() {
+        List<UserDto> users = userService.getAllUsers();
+        Set<UserWithTeamsDto> usersWithTeams = new HashSet<>();
+
+        for (UserDto user: users){
+            Set<Team> teams = teamService.getTeamsForUser(UUID.fromString(user.getId()));
```

Comment:
While this does work - what this won't do is explode out a CASE_ADMIN to show there in every team. But I think thats okay?

Reply:
I think that's ok for the purposes this will be used for, we could potentially look at for the users role and do something specifically for CASE_ADMINS if needed in the future

Reply:
Agreed - just one to note down.